### PR TITLE
fix: add `/metadata/v1` prefix to all routes

### PR DESCRIPTION
**Pull request by rafaelcr**
Also add the `/metadata` prefix with the same effect

### CODE REVIEW

**Comment by CharlieC3** [comment]
> Also, rewrite /metadata/* URLs to /metadata/v1/*

Based on @saralab's [comment](https://github.com/hirosystems/token-metadata-service/issues/99#issuecomment-1419355439), I'm not sure this scenarios calls for any HTTP path rewriting. For example, `/metadata/cool/function` should resolve as-is and so should `/metadata/v1/cool/function`. No rewrites should occur and they should both perform the same action while `v1` is the latest available versioned API provided by the TMS.

Months or years from now when API version `v2` comes out and `v1` is officially removed after an appropriate deprecation period, `/metadata/v1/cool/function` would redirect to `/metadata/v2/cool/function` or `/metadata/cool/function`. So I don't believe we'll need any URL redirects for some time (I'm aware rewrites are different from redirects, but figure I'd cover both since they're a bit tangential).

**Comment by rafaelcr** [comment]
Gotcha, I'll make that change @CharlieC3

File: src/api/util/helpers.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { IncomingMessage } from 'http';
```
Comment (zone117x) [inline src/api/util/helpers.ts:1]: Unintentional import?

Comment (rafaelcr) [reply src/api/util/helpers.ts]: Yep, done

**Comment by rafaelcr** [comment]
Done @CharlieC3 can you take another look?

File: tests/ft.test.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ describe('FT routes', () => {
   test('token not found', async () => {
     const response = await fastify.inject({
       method: 'GET',
-      url: '/ft/SP2SYHR84SDJJDK8M09HFS4KBFXPPCX9H7RZ9YVTS.hello-world',
+      url: '/metadata/v1/ft/SP2SYHR84SDJJDK8M09HFS4KBFXPPCX9H7RZ9YVTS.hello-world',
```
Comment (CharlieC3) [inline tests/ft.test.ts:42]: We should probably test the non-versioned endpoints as well to ensure they always match the response received from the latest versioned endpoints.

Comment (rafaelcr) [reply tests/ft.test.ts:42]: Done

**Pull request merged**